#! /bin/sh /usr/share/dpatch/dpatch-run
## DP: fix library path

@DPATCH@

--- mono-zeroconf/src/Mono.Zeroconf/Mono.Zeroconf.Providers/ProviderFactory.cs.old	2007-12-20 05:53:53.000000000 +0100
+++ mono-zeroconf/src/Mono.Zeroconf/Mono.Zeroconf.Providers/ProviderFactory.cs	2007-12-20 05:54:11.000000000 +0100
@@ -86,8 +86,9 @@
                 for(int i = 0; i < path_parts.Length - 4; i++) {
                     new_path = Path.Combine(new_path, path_parts[i]);
                 }
-                
-                directories.Add(Path.Combine(new_path, "mono-zeroconf"));
+
+                new_path = Path.Combine(new_path, "cli");
+                directories.Add(Path.Combine(new_path, "Mono.Zeroconf-1.0"));
             }
             
             foreach(string directory in directories) {
